addons/survey/views/survey_templates_user_input_session.xml

```diff
@@ -107,21 +107,16 @@
                     </div>
                 </div>
             </div>
-            <div class="o_container_small px-4 pb-3 pt96 d-flex flex-column o_survey_session_manage_container">
```

Comment:
Hello @aath-odoo :slightly_smiling_face: 

You moved the "next" navigation button but we're in stable so we should minimize any modification on the html tags structure.
I also feel like changing the position fixed to a d-flex might push the graph to one side of the screen. It won't be centered anymore and its position will be depending on the button text width :thinking: :confused: 

I tested locally and I think I got something working :slightly_smiling_face: 
What about:
- removing the `o_container_small` here (as you did)
- setting the `o_container_small` class to the "Thank you" div just underneath (as you also did) 
- keeping your `d-flex justify-content-center` change on the title as well
- changing the `px-4` here to `px-sm-4` to prevent an horizontal scrollbar on lower screen sizes

Finally, to prevent the overflow of the navigation button on the chart, you could just add:
`max-width: 10%` on the `.o_survey_session_navigation` class in the scss file.
We could also hide the "next" button label on small screen sizes with a `d-none d-md-inline` or something :slightly_smiling_face: 

With all that, we got the page looking like so:
https://tinyurl.com/2xlz2nkw 

What do you think ? :thinking:  :slightly_smiling_face: 
Could you look into my proposition and tell me if I missed something :pray: 

Thanks !

Reply:
Hello @amdi-odoo  I tried your suggested approach, and it seems to work well. Thanks! If it's functioning properly without needing changes to the HTML structure, we can proceed with it. Thanks again! :)